### PR TITLE
CORE-20434: Remove rest and crypto cluster-level tenants from the docs

### PR DESCRIPTION
**Pull request by anien**
We have now removed rest and crypto cluster-level tenants. Update our docs to reflect this.

### CODE REVIEW

File: components/membership/membership-rest/src/main/kotlin/net/corda/membership/rest/v1/CertificateRestResource.kt [hunk 1]
```diff
@@ -285,8 +285,8 @@ interface CertificateRestResource : RestResource {
     )
     fun generateCsr(
         @RestPathParameter(
-            description = "Can either be a holding identity ID, the value 'p2p' for a cluster-level" +
```
Comment (vkolomeyko) [inline components/membership/membership-rest/src/main/kotlin/net/corda/membership/rest/v1/CertificateRestResource.kt:288]: There are few instances in the file where it says: `'rest-tls' for a TLS certificate to be used in REST communication` - I do believe this should be removed.

File: components/membership/membership-rest/src/main/kotlin/net/corda/membership/rest/v1/CertificatesRestResource.kt [hunk 2]
```diff
@@ -274,8 +274,8 @@ interface CertificatesRestResource : RestResource {
      * subjectAlternativeNames = ["localhost"], contextMap = {"signatureSpec": "SHA256withECDSA"})
      * ```
      *
-     * @param tenantId Can either be a holding identity ID, the value 'p2p' for a cluster-level tenant of the P2P
```
Comment (vkolomeyko) [inline components/membership/membership-rest/src/main/kotlin/net/corda/membership/rest/v1/CertificatesRestResource.kt:277]: Same for this file - I think we should remove: `'rest-tls' for a TLS certificate to be used in REST communication`

File: components/membership/membership-rest/src/main/kotlin/net/corda/membership/rest/v1/CertificateRestResource.kt (hunk 1 above)
Comment (anien) [reply components/membership/membership-rest/src/main/kotlin/net/corda/membership/rest/v1/CertificateRestResource.kt:288]: This is baked in avro schemas, which we haven't changed. If we want to remove this certificate usage, I would suggest a separate ticket as it's a bigger change and not related to this ticket I believe.

**Review by Sakpal — COMMENTED** [review]
LGTM

File: components/membership/membership-rest/src/main/kotlin/net/corda/membership/rest/v1/CertificateRestResource.kt (hunk 1 above)
Comment (vkolomeyko) [reply components/membership/membership-rest/src/main/kotlin/net/corda/membership/rest/v1/CertificateRestResource.kt:288]: Sure - we can do this task a part of a separate Jira. @anien - could you raise this ticket, please?

Comment (anien) [reply components/membership/membership-rest/src/main/kotlin/net/corda/membership/rest/v1/CertificateRestResource.kt:288]: Ticket raised: https://r3-cev.atlassian.net/browse/CORE-20583

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Review by williamvigorr3 — APPROVED** [review]
LGTM

**Pull request merged**